src/server/vendor/vendor.json

```diff
@@ -2,6 +2,18 @@
 	"comment": "",
 	"ignore": "test",
 	"package": [
+		{
```

Comment:
Not sure how these annotations got added. Were these packages added but the metadata not recorded? If they're added, the metadata should be here. If they're removed, we can remove these stubs as well.

Reply:
I think they may be dependencies of the new client. Basically, the commands I ran were:
```
govendor fetch 'k8s.io/client-go/...@v7.0.0'
govendor fetch +m  # I saw packages were missing, so I assume this stuff was added here
govendor remove +u
```